rpc/sigpipe.go

```diff
@@ -0,0 +1,9 @@
+// +build !darwin
+
+package rpc
+
+import "net"
+
+func DisableSigPipe(c net.Conn) error {
+	return nil
```

Comment:
> Right now I am just setting this on every connection we make using the library

Then why even have an option?  Just always do it on darwin in the rpc library, then there's no need to change the callers and there's no confusion, the story can be contained entirely in the comments in this package.